dhall-json/tasty/data/mismatchMessage0.golden

```diff
@@ -0,0 +1,8 @@
+[1;31m
```

Comment:
How about moving the new tests into a separate `error` folder within the `data` directory?